test/integration/server-test.ts

```diff
@@ -98,6 +97,11 @@ test("POST / with push event payload (request.body already parsed)", (t) => {
   const server = http.createServer((req, res) => {
     req.once("data", (chunk) => dataChunks.push(chunk));
     req.once("end", () => {
+      /*
+       * @ts-ignore: Until getPayload @ts-ignore is fixed
+       * (https://github.com/dominguezcelada/webhooks.js/blob/b2162f4489efb9ee828cfd57e82bf98d12503c4b/src/middleware/get-payload.ts#L7-L8)
+       */
+      //@ts-ignore
       req.body = JSON.parse(Buffer.concat(dataChunks).toString());
```

Comment:
ts-ignored until this is fixed:

https://github.com/dominguezcelada/webhooks.js/blob/b2162f4489efb9ee828cfd57e82bf98d12503c4b/src/middleware/get-payload.ts#L7-L8